client/post-editor/media-modal/detail/index.jsx

```diff
@@ -16,7 +16,7 @@ var DetailItem = require( './detail-item' ),
 import { ModalViews } from 'state/ui/media-modal/constants';
 import { setEditorMediaModalView } from 'state/ui/editor/actions';
 
-const EditorMediaModalDetail = React.createClass( {
+export const EditorMediaModalDetail = React.createClass( {
```

Comment:
Oh gosh! It didn't even occur for me to check!
Funny the tests passed though :/
I will definitely fix that!